### ISSUE TITLE
Undertubulated long bones

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7466]
**Preferred term label:**

**Synonyms**
Undertubulation of diaphyses

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:601559
PMID:14740318 8/8

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also 
PMID: 24988918  6/6   OMIM:601559

**Comment by pnrobinson** [comment]
see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7311375/

**Comment by pnrobinson** [comment]
@azankl -- is this the same as Diaphyseal thickening HP:0005019

**Comment by azankl** [comment]
@pnrobinson I'd say yes, but my preferred label for HP:0005019 would be 'diaphyseal widening', with 'undertubulated long bones' and 'undertubulation of diaphyses' as synonyms. 'Diaphyseal thickening' sounds like thickening of the diaphyseal cortex to me, which is HP:0005791, so would avoid 'diaphyseal thickening' as its ambiguous (it could refer to HP:0005019 or HP:0005791).

**Comment by pnrobinson** [comment]
@azankl 
thanks!
I have updated the definition and added the various synonyms.
